src/Formik.tsx

```diff
@@ -503,6 +508,45 @@ export class Formik<Values = {}, ExtraProps = {}> extends React.Component<
     }));
   };
 
+  setFieldFocused = (field: string, focused: boolean = false) => {
+    this.setState(prevState => ({
+      ...prevState,
+      focused: setIn(prevState.focused, field, focused),
+    }));
+  };
+
+  handleFocus = (eventOrString: any): void | ((e: any) => void) => {
+    const executeFocus = (e: any, path?: string) => {
+      if (e.persist) {
+        e.persist();
+      }
+      const { name, id, outerHTML } = e.target;
+      const field = path ? path : name ? name : id;
+
+      if (!field && process.env.NODE_ENV !== 'production') {
+        warnAboutMissingIdentifier({
+          htmlContent: outerHTML,
+          documentationAnchorLink: 'handlefocus-e-any--void',
+          handlerName: 'handleFocus',
+        });
+      }
+
+      this.setState(prevState => ({
+        focused: setIn(prevState.focused, field, true),
```

Comment:
Only one field at the time can be focused. So all the rest of the fields should be turned to false